src/System.Security.Cryptography.Encoding/tests/Asn1/Writer/Asn1WriterTests.cs

```diff
@@ -12,8 +12,11 @@ public abstract partial class Asn1WriterTests : Asn1ReaderTests
     {
         internal static void Verify(AsnWriter writer, string expectedHex)
         {
+            int expectedSize = writer.GetEncodedLength();
+
             byte[] encoded = writer.Encode();
             Assert.Equal(expectedHex, encoded.ByteArrayToHex());
+            Assert.Equal(encoded.Length, expectedSize);
```

Comment:
The expected value goes on the left, so either the parameter order should be flipped or the local should be renamed to not be "expected".

If `expectedSize` were 4, and `encoded.Length` were 3 this would currently print

```
Assert.Equals
    Expected: 3
      Actual: 4
```

which would then be confusing since "expectedSize" was the one that was 4.